schedule/yast/sle/guided_xfs/guided_xfs_x86_64.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name:           guided_xfs
+description:    >
+  Guided Partitioning installation with xfs filesystem.
+vars:
+  YUI_REST_API: 1
+schedule:
+  extension_module_selection:
+    - installation/module_registration/register_module_desktop
+  system_role:
+    - installation/system_role/accept_selected_role_SLES_with_GNOME
+  guided_partitioning:
+    - installation/partitioning/select_guided_setup
+    - installation/partitioning/guided_setup/accept_default_part_scheme
+    - installation/partitioning/guided_setup/select_filesystem_option_xfs
+    - installation/partitioning/accept_proposed_layout
```

Comment:
This module is already scheduled in [suggested_partitioning](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/60b576bbb7a8e930486b9754bfabf66dce07c0ed/schedule/yast/sle/flows/default.yaml#L27). Also, there is no guided_partitioning in the default file, it's a bug that it works in my opinion.

Reply:
changed.